submissions/Esebame1/Week_1_solution.py

```diff
@@ -1,11 +1,9 @@
 for a in range (1,100):
   if a%3==0 and a%5==0:
     print('Fizzbuzz\n')
-     continue
   elif a%3==0:
     print('Fizz\n')
-     continue
   elif a%5==0:
     print('Buzz\n')
-     continue
+     else:
   print (str(a)+'\n')
```

Comment:
Indenting seems to be off on this line and the one before it... This might result in a SyntaxError.
Can you check that and make changes?